### DISCUSSION TITLE
Is there a planned release date for a Toggle component?

### DISCUSSION THREAD

**Discussion by sang-yeo** [twilio-labs/paste#694] [Q&A] [answered]
Is there a timeline for a new Toggle component? Few Studio Widgets have used old console toggle component and wondering new component will be available before the new console udl.

**Comment by kate-porter** [comment]
Hey @sang-yeo! Checking in with the team, I'll get back to ya soon.

**Comment by kate-porter** [comment, marked as answer]
Hey @sang-yeo - so, I'm not sure what your timeline is, but we've got a toggle component _very tentatively_ on our roadmap for release in November. You can always find our roadmap [here](https://paste.twilio.design/roadmap), but with the caveat that the further out the date, the more tentative the list of components in that release.

If you need something before then, I'd recommend first considering if there is a different component that might be able to suit your needs (as an example, a single checkbox can replace the functionality of a toggle in cases where you need on/off functionality). If that's not an option, you may need to stick with the old console toggle for now, or you might try partnering with your engineers to see if you could create a custom toggle component using the Box primitive in Paste.

Sarah is also working on setting up roadmap reviews with all the BUs over the next few weeks, so be sure to mention that you need a toggle component, and that'll help us prioritize it.

**Reply by andioneto** [reply]
@sang-yeo I wrapped the legacy toggle in Paste styles, if you're curious to see that. Seems to be working nicely as a short term replacement. Feel free to message me on Slack and I can provide more info.
